ember-cli-build.js

```diff
@@ -9,6 +9,20 @@ module.exports = function (defaults) {
   };
 
   var app = new EmberApp(defaults, {
+    prember: {
+      urls: [
+        '/',
+        '/timetable',
+        '/about',
+        '/subscribe',
+        '/dj-inquiry',
+        '/coc',
+        '/djs',
+        '/chat',
+        '/sign-up',
```

Comment:
I just did this quickly to experiment, but in reality we have pages for every show that will need to be crawled. 
According to the prember docs we can use a special URL discovery function.
https://github.com/ef4/prember#using-a-custom-url-discovery-function

Is this even possible ? Will we have to crawl the entire schedule back to the start of the station? :laughing:

Reply:
Might be some info in here. https://github.com/ef4/prember/issues/40